src/pynwb/data/nwb.file.yaml

```diff
@@ -235,6 +235,10 @@ groups:
           dtype: text
           doc: Value is 'A recording device e.g. amplifier'.
           value: A recording device e.g. amplifier
+        datasets:
```

Comment:
Why is this a dataset and not an attribute? I would not expect this to be changed once it is written, so shouldn't it be an attribute (https://github.com/NeurodataWithoutBorders/nwb-schema/issues/48)? Making this required would be a schema-breaking change, so I would like to make this optional.